### ISSUE TITLE
Infill, Top & Bottom not filling

### ISSUE THREAD

**Issue by pierremilcent** [Ultimaker/Cura#12057]
### Application Version

5.0 Beta

### Platform

MacOS M1

### Printer

Artillery Sidewinder X2

### Reproduction steps

i am not able to get the same result with Cura 5 and Cura 4.13.1 for the same part. Cura 5 does not allow me to have infill done, it keeps printing 2 walls and nothing inside.

What should i check in Cura 5 parameters that influence this behaviour ? 

Thanks
<img width="1426" alt="Capture d’écran 2022-05-02 à 11 41 23" src="https://user-images.githubusercontent.com/38214885/166216347-b180476b-630d-4328-b565-9d5c8c20e72c.png">
<img width="1428" alt="Capture d’écran 2022-05-02 à 11 41 32" src="https://user-images.githubusercontent.com/38214885/166216366-4c318ec0-a545-42e6-921a-2a9f8e548d59.png">
.

### Actual results

<img width="1428" alt="Capture d’écran 2022-05-02 à 11 41 32" src="https://user-images.githubusercontent.com/38214885/166216399-28c3ce2a-e5fc-4128-906a-648ef53dac6b.png">

### Expected results

<img width="1426" alt="Capture d’écran 2022-05-02 à 11 41 23" src="https://user-images.githubusercontent.com/38214885/166216442-e7d57396-e7a6-4a89-8029-227295741e69.png">

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

none

**Comment by fvrmr** [comment]
Hi @pierremilcent thank you for your report.
Could you check if the infill is enabled in preview mode?
![Capture(344)](https://user-images.githubusercontent.com/77000924/166238842-7ca4b9c6-462f-49a0-a7c3-4773d3134d40.PNG)

If that is not the case, please share your project file (file - save project) as a .zip here on Github.

**Comment by pierremilcent** [comment]
Yes it is checked. 2 screenshots to illustrate. I also add 3mf files for both cura versions.
<img width="714" alt="Capture d’écran 2022-05-02 à 19 05 29" src="https://user-images.githubusercontent.com/38214885/166292569-95db8170-3093-4bbf-b5e9-429b7976b511.png">
<img width="718" alt="Capture d’écran 2022-05-02 à 19 05 38" src="https://user-images.githubusercontent.com/38214885/166292575-58035b09-dff0-4719-b22f-ddac866e59ba.png">
[3mf.zip](https://github.com/Ultimaker/Cura/files/8604861/3mf.zip)
